25-minimize.ks

```diff
@@ -64,14 +62,20 @@ echo fdi > /etc/hostname
 echo " * locking root account"
 passwd -l root
 
+echo " * store list of packages sorted by size"
+rpm -qa --queryformat '%{SIZE} %{NAME}\n' | sort -n -r > /usr/PACKAGES-LIST
+
 echo " * cleaning up yum cache and removing rpm database"
-rpm -qa > /usr/packages_list
```

Comment:
Actually, the name was referenced from `discovery-debug` therefore I fixed this and pushed.